lib/etna/http_codes.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Etna
```

Comment:
So the last thing I want to ask, is there no way to structure and generalize these hard coded http codes so they can also be used in the big switch case in `responder.rb`(line 21)?

My reason is simply that if there's a change(or mistake) in the the way an API has implemented the spec, then you have to maintain it in two places. Here and in that big old switch case.